test/components/form/batch-async-command.js

```diff
@@ -51,18 +50,22 @@ test('collect multiple errors', function (t) {
       }
 
       process.nextTick(() => {
-        command.setProps(item);
+        command.update();
       });
     };
 
+    const cleanup = () => {
+      console.log('Cleanup called');
+    };
+
     const item = {
       text: 'Example',
       action: action,
       state: {},
       confirm: noop
     };
 
-    const command = shallow(
+    const command = mount(
```

Comment:
Initially when using the test file (before Jenny mentioned that .instance can't be used for functional components) I did some research on the difference between the different rendering styles since I figured that was what the issue was. I found that mount does deeper work in terms of testing components. Mount is also more reliable and secure when looking at debugging as it breaks when there's a failure while shallow continues to render and pass the parent component even if there is a break in another component. Mount also tests full lifecycles of components.